src/main/java/mezz/jei/gui/recipes/RecipesGui.java

```diff
@@ -467,9 +474,9 @@ private void updateLayout() {
 
 		logic.setRecipesPerPage(recipesPerPage);
 
-		title = recipeCategory.getTitle();
+		title = recipeCategory.getTitleAsTextComponent();
```

Comment:
Does this allow font formatting like bold or colors? That's ok for tooltips, but we should strip that out of things like titles that are displayed in the JEI GUIs to enforce a more consistent look.